app/routers/tasks.py

```diff
@@ -47,7 +51,7 @@
 
 from app.crud.audio_transcription import create_audio_transcription
 from sqlalchemy.orm import Session
-from app.deps import get_db
```

Comment:
@katg999 you did remove this import but actually this import is required for the `/stt` endpoint as dependency.

Reply:
Some of these changes I had tried to rectify them after the first commit but then I dont know why they have not been reflected here.